Samples.xcodeproj/project.pbxproj

```diff
@@ -615,6 +617,8 @@
 			dstPath = "";
 			dstSubfolderSpec = 7;
 			files = (
+				88E52E6C2DC960EA00F48409 /* ApplyFunctionToRasterFromServiceView.swift in Copy Source Code Files */,
+				00FA4E592DBC1786008A34CF /* AddRasterFromGeopackageView.swift in Copy Source Code Files */,
```

Comment:
This line shouldn't be here. Remember to always have Xcode re-save the project file (eg. by renaming or rearranging a file and then undoing) after making manual modifications to it. When you do that for this project, this line will go away.